pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -219,7 +231,7 @@ func (c *ClientStartConfig) makeNodeConfig(out io.Writer, masterConfigDir string
 	if err != nil {
 		return "", "", fmt.Errorf("error creating node config: %v", err)
 	}
-	kubeDNSConfigDir, err := container.MakeNodeConfig(c.GetDockerClient(out), imageRunningHelper.New(), out)
+	kubeDNSConfigDir, err := container.MakeKubeDNSConfig(c.GetDockerClient(out), imageRunningHelper.New(), out)
 	if err != nil {
 		return "", "", fmt.Errorf("error creating node config: %v", err)
 	}
```

Comment:
Below this somewhere, there should be an `if c.WriteConfig{ //copy the content to the correct locations and exit}`